ddp-workspace/projects/ddp-sdk/src/lib/components/subjectPanel/subjectPanel.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component, Inject, Input, OnInit } from '@angular/core';
+import { SessionMementoService } from '../../services/sessionMemento.service';
+import { ConfigurationService } from '../../services/configuration.service';
+import { InvitationPipe } from '../../pipes/invitationFormatter.pipe';
+import { SearchParticipant } from '../../models/searchParticipant';
+
+interface SubjectField {
```

Comment:
#optional
suggest moving the interface to models folder

Reply:
I'd prefer to keep it here since it's not a global entity interface but rather a helper interface for component view